core/common/modules/connect/apps/base-app.php

```diff
@@ -685,10 +689,10 @@ public function get_site_key() {
 	protected function redirect_to_remote_authorize_url() {
 		switch ( $this->auth_mode ) {
 			case 'cli':
-				$this->get_app_token_from_cli_token( $_REQUEST['token'] );
+				$this->get_app_token_from_cli_token( Utils::get_super_global_value( $_REQUEST, 'token' ) ); //phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce verification is not required here.
 				return;
 			default:
-				wp_redirect( $this->get_remote_authorize_url() );
+				wp_redirect( $this->get_remote_authorize_url() ); //phpcs:ignore WordPress.Security.SafeRedirect.wp_redirect_wp_redirect -- Safe redirect is used here.
```

Comment:
It may be filtered, maybe use `wp_safe_redirect` and add the remote host as a safe host

Reply:
Done, nice to know.